<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTextEdit" name="textRecv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="LED1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="LED2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="LED3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="LED4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发送窗口</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>517</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM4</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>bps</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpen">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSend">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
